Sources/ComposableArchitecture/Effect.swift

```diff
@@ -591,3 +607,63 @@ extension Effect {
     }
   }
 }
+
+// MARK: - Transitory Deprecation
+
```

Comment:
We don't use `MARK: -` very often, so let's get to the point earlier :smile:

```suggestion
```

Reply:
There was one above, so I reproduced. I'll clean the whole file

Reply:
@stephencelis, Ah, you performed the change. I'll leave other existing `MARK:` for now.